src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -288,7 +293,7 @@ div.sf-toolbar  .sf-toolbar-block .sf-toolbar-info-piece.sf-toolbar-info-php-ext
     color: var(--sf-toolbar-yellow-800);
 }
 
-.sf-toolbar-block.sf-toolbar-status-green {
+.sf-toolbar-block.sf-toolbar-status-green:not(:hover) {
     background-color: #059669;
```

Comment:
@javiereguiluz this green status is not consistent with the red and yellow statuses (that use a bottom colored area only). Is it expected ?

Reply:
It's not expected ... in fact, I think there should be no "green status" in the toolbar items. OK is the default status of all items, so there's no need to show it as green ever (except for the status code in the special request panel).

Reply:
Removing the green status class would be some kind of BC break, so maybe I can make it work as the red & yellow ones?

Reply:
Yes, that's a good idea. We won't use it, but others may.